src/main/msp/msp.c

```diff
@@ -3646,6 +3652,11 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         break;
 #endif
 
+#if defined(USE_RANGEFINDER_MSP)
+    case MSP2_SENSOR_RANGEFINDER:
+        mspRangefinderReceiveNewData(sbufPtr(src));
```

Comment:
(packet length is not checked)
Relying on `struct {} packed` is quite fragile, it will make future improvement of rangefinder code harder.